styles/globals.css

```diff
@@ -29,7 +63,7 @@ a {
 }
 
 .max_width {
-  max-width: 1240px;
+  max-width: 1180px;
```

Comment:
It looks like this max-width is defined in multiple places, maybe it could be in it's own style

Reply:
How do you mean sir

Reply:
I don't know :)